doc/user/content/integrations/cube.md

```diff
@@ -3,6 +3,7 @@ title: "Cube"
 description: "How to connect a Cube deployment to Materialize"
 aliases:
   /third-party/cube/
+draft: true
```

Comment:
Not sure what to do about this page.  It wasn't in the nav ... so, just marked as draft.  We can either rm it or surface it.